translations/en-us.yml

```diff
@@ -800,7 +800,7 @@ institutions:
             preprints: Preprints
         content-placeholder: Content coming soon # Delete this eventually pls
         title: '{institutionName} Dashboard'
-        last_update: 'Updated every 24 hours'
+        download_past_reports_label: Reports
```

Comment:
(text nitpick) seems less than ideal that the text shown to the user is even less descriptive than the variable names

Reply:
Yea, I'll propose a more descriptive label for this